apps/dotcms-ui/src/app/api/services/dot-license/dot-license.service.ts

```diff
@@ -9,6 +9,13 @@ export interface DotUnlicensedPortletData {
     url: string;
 }
 
+interface License {
```

Comment:
```suggestion
interface DotLicense {
```

We prepend our types like this.

Also... put this type in the `dotcms-models` library, looks very reusable in other apps and libs.